packages/platform-sdk-profiles/src/profiles/authenticator.ts

```diff
@@ -11,6 +12,8 @@ export class Authenticator {
 
 	public setPassword(password: string): void {
 		this.#profile.settings().set(ProfileSetting.Password, Bcrypt.hash(password));
+
+		MemoryPassword.set(this.#profile, password);
```

Comment:
Needs a test to ensure that the in-memory password has been set when this method is called.

Reply:
Modified those in `env.test.ts` and `profile-repository` to check this, but I 'll add a separate one for authenticator